screens/AppStack/CreateReminderScreen.tsx

```diff
@@ -34,6 +39,34 @@ const days = [
   { id: 6, name: 'Saturday' },
 ]
 
+type Time = {
+  id: Number
+  value: string
+}
+
+const times: Time[] = [
+  { id: 0, value: '05:00 am' },
+  { id: 1, value: '06:00 am' },
+  { id: 2, value: '07:00 am' },
+  { id: 3, value: '08:00 am' },
+  { id: 4, value: '09:00 am' },
+  { id: 5, value: '10:00 am' },
+  { id: 6, value: '11:00 am' },
+  { id: 7, value: '12:00 pm' },
+  { id: 8, value: '01:00 pm' },
+  { id: 9, value: '02:00 pm' },
+  { id: 10, value: '03:00 pm' },
+  { id: 11, value: '04:00 pm' },
+  { id: 12, value: '05:00 pm' },
+  { id: 13, value: '06:00 pm' },
+  { id: 14, value: '07:00 pm' },
+  { id: 15, value: '08:00 pm' },
+  { id: 16, value: '09:00 pm' },
+  { id: 17, value: '10:00 pm' },
+  { id: 18, value: '11:00 pm' },
+  { id: 19, value: '12:00 am' },
+]
```

Comment:
To make this simpler, I think we want to dropdowns. One for the hour and one for the minutes. Then we can have hours `0-24` and 5 minutes increments. If we really decide `0-24` is too much for a dropdown, we can have an AM/PM dropdown too. 🙂

Reply:
In fact, let's use this: https://www.npmjs.com/package/react-native-modal-datetime-picker